pandas/_libs/ops_dispatch.pyi

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+REVERSED_NAMES: dict[str, str]
+UFUNC_ALIASES: dict[str, str]
+DISPATCHED_UFUNCS: set[str]
```

Comment:
These are not accessed from python code.

so I don't think is covered by https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#what-to-include

and from https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#conventions

> The general rule is that they should be as concise as possible.

so I think we should perhaps not include these.

once we activate `ignore_missing_imports = False`, we'll know if we missed anything.

---

side note: for the stub files we could consider activating more of the strictness flags for incomplete function annotations and dynamic typing etc.